tests/unittest/_torch/visual_gen/test_wan22_nvfp4_fusion_integration.py

```
@@ -0,0 +1,498 @@
+    not _CKPT_PATH,
+    reason="Wan 2.2 NVFP4 checkpoint not found. Set WAN22_T2V_NVFP4_PATH or place "
+    "the model at /models/Wan2.2-T2V-A14B-Diffusers-NVFP4.",
+)
+
+
+def _has_sm100() -> bool:
+    if not torch.cuda.is_available():
+        return False
+    try:
+        sm = get_sm_version()
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)